backend/src/lib/controllers/user-ctrl.ts

```diff
@@ -51,6 +54,24 @@ async function addUsers(req: Request, res: Response) {
   }
 }
 
+async function removeUsers(req: Request, res: Response) {
+  const validationResult = validate(req.body, RemoveUsersSchema);
+  if (!validationResult.valid) {
+    res.status(400);
+    logger.warn("Invalid request to remove users %o", validationResult.errors);
+    return res.send(validationResult.toString());
+  }
```

Comment:
@miguelpdiaz8 Let's start doing validation in the Backend using jsonschema like this. What do you think?

Reply:
@trietlu

Reply:
SGTM

Reply:
@ferdingler this is great!